internal/feature_flags/test/register_fake.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package test
+
+import (
+	"github.com/newrelic/infrastructure-agent/pkg/backend/identityapi"
+)
+
+type fakeRegisterClient struct {
```

Comment:
I don't see this being used

Reply:
It was originally when `agent.go` took a instance of `RegisterClient`. Will remove as we can always add it back if needed.

Reply:
Deleted.